_src/_sass/_layout.scss

```diff
@@ -117,6 +119,8 @@
     width: 40em;
     // temporary fix
     display: table;
+    // debug
+    background-color: orchid;
```

Comment:
The only change that has an effect in this file.
The rest is just formatting, maybe revert those. (in this file)

Reply:
Will do.

Reply:
will do